### PR TITLE
Text Entered by User Should be Line breaked as it is he entered

### PR DESCRIPTION
**Pull request by Gurmeet-Singh657**
Resolved Issue-Number-348
Text Entered by User will be sent as it is to backend and fetched by preserving Line Breaks
Please merge my pull request

image

### CODE REVIEW

**Review by narayan954 — CHANGES_REQUESTED** [review]
Hi @Gurmeet-Singh657 are you sure your pr is in sync with this repo, as I can see a lots of feature are being overwritten in this pull request, Please fix that

**Comment by Gurmeet-Singh657** [comment]
I have Resolved the Changes. Please merge my Pull Request !!

**Review by narayan954 — CHANGES_REQUESTED** [review]
Kindly doubleCheck your changes, why are you removing so many features which are already present in the application

**Comment by Gurmeet-Singh657** [comment]
@narayan954 If possible can you plz help me in resolving this !! Because I think there is some inconsistency in my forked repo

**Comment by narayan954** [comment]
> @narayan954 If possible can you plz help me in resolving this !! Because I think there is some inconsistency in my forked repo

You may force update your fork with my master branch

File: src/components/Post.jsx [hunk 1]
```diff
@@ -426,7 +426,7 @@ function Post(prop) {
               </>
             ) : (
               <>
-                <p className="post_caption">{caption}</p>
+                <p style={{ whiteSpace: 'pre-line' }} className="post_caption">{caption}</p>
```
Comment (narayan954) [inline src/components/Post.jsx:429]: instead of adding inline styles, can we put these styles where they belong in index.css?

**Review by narayan954 — CHANGES_REQUESTED** [review]
minor change requested

**Review by narayan954 — APPROVED** [review]
lgtm!

**Pull request merged**